charts/zabbix/values.yaml

```diff
@@ -631,25 +643,41 @@ karpenter:
   # Operators { In, NotIn, Exists, DoesNotExist, Gt, and Lt } are supported.
   # https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#operators
   requirements:
-    - key: "karpenter.k8s.aws/instance-category"
-      operator: In
-      values: ["c", "m", "r"]
-    - key: "karpenter.k8s.aws/instance-cpu"
-      operator: In
-      values: ["2", "4", "8", "16", "32"]
-    - key: "kubernetes.io/arch"
-      operator: In
-      values: ["amd64"]
-    - key: kubernetes.io/os
-      operator: In
-      values: ["linux"]
-    - key: "karpenter.sh/capacity-type"
-      operator: In
-      values: ["spot", "on-demand"]
+  - key: "karpenter.k8s.aws/instance-category"
+    operator: In
+    values: ["c", "m", "r"]
+  - key: "karpenter.k8s.aws/instance-cpu"
+    operator: In
+    values: ["2", "4", "8", "16", "32"]
+  - key: "kubernetes.io/arch"
+    operator: In
+    values: ["amd64"]
+  - key: kubernetes.io/os
+    operator: In
+    values: ["linux"]
+  - key: "karpenter.sh/capacity-type"
+    operator: In
+    values: ["spot", "on-demand"]
   # -- Optional, configures IMDS for the instance
   metadataOptions:
     httpEndpoint: enabled
     httpProtocolIPv6: disabled
     httpPutResponseHopLimit: 2
     httpTokens: required
 
+serviceAccount:
+  # Specifies whether a service account should be created.
```

Comment:
Hi @syphernl1

Can you change this line?

`# -- Specifies whether a service account should be created.`